source/includes/azure/_instances.md

```diff
@@ -70,6 +76,12 @@ Attributes | &nbsp;
 `privateIp`<br/>*string* | The private ip address assigned to the instance
 `internalFqdn`<br/>*string* | The internal FQDN assigned to the instance
 `macAddress`<br/>*string* | The MAC address assigned to the instance
+`subnetName`<br/>*string* | The name of the subnet that the instance is part of
+`networkName`<br/>*string* | The name of the network that the instance is part of
+`supportsPremiumIO`<br/>*boolean* | If the instance supports Premium IO. This is fixed by the machine type.
+`supportsUltraIO`<br/>*boolean* | If the instance supports Ultra IO. This is fixed by the machine type.
+`attachableDiskSlots`<br/>*integer* | The number of data disk that can be attached to the instance. This is fixed by the machine type.
+`usedLuns`<br/>*array* | An array of all the LUNs already used for the data disks.
```

Comment:
inconsistent period use with `supportsPremiumIO`, `supportsUltraIO`, `attachableDiskSlots`, and `usedLuns` when compared with other pre-existing ones such as `networkInterfaces` which doesnt end in a period even with 2 sentences

Reply:
7d86cd5d2eaed27328435a666935632ba6db7c88